packages/schemas/src/player/gamemodes/woolgames/class.ts

```diff
@@ -0,0 +1,89 @@
+import { ratio } from '@statsify/math';
+import { APIData } from '@statsify/util';
+import { Field } from '../../../metadata';
+
+export class WoolWarsClass {
+  @Field()
+  public wins: number;
+
+  @Field()
+  public gamesPlayed: number;
+
+  @Field()
+  public losses: number;
+
+  @Field()
+  public kills: number;
+
+  @Field()
+  public deaths: number;
+
+  @Field()
+  public kdr: number;
+
+  @Field()
+  public assists: number;
+
+  @Field()
+  public powerups: number;
+
+  @Field()
+  public blocksBroken: number;
+
+  @Field()
+  public woolPlaced: number;
+
+  public constructor(data: APIData) {
+    this.wins = data?.wins ?? 0;
```

Comment:
you should be able to remove a lot of these ``?? 0`. Serialization/Deserialization will take care of it.

Reply:
this is untrue :(!